package.json

```diff
@@ -6,10 +6,10 @@
   "private": true,
   "dependencies": {
     "aos": "^2.3.3",
-    "gatsby": "next",
-    "gatsby-link": "^1.6.46",
+    "gatsby": "^2.0.8",
+    "gatsby-link": "^2.0.1",
     "gatsby-plugin-catch-links": "^1.0.24",
-    "gatsby-plugin-react-helmet": "next",
+    "gatsby-plugin-react-helmet": "3.0.0",
     "gatsby-plugin-sass": "next",
     "gatsby-plugin-sharp": "next",
```

Comment:
Worth potentially trying to mark these two to get them off `next` as well?

Reply:
I'll spend a few minutes - hopefully that's all it takes. Let's see if they're little monsters or not. I wanted to keep this update minimal for now, so I could get Ramiro up and running.